### DISCUSSION TITLE
Documentation: List all keyboard shortcuts

### DISCUSSION THREAD

**Discussion by eduardoalonsoalbella** [craftcms/docs#289] [Ideas] [answered]
Maybe it already exists, but haven't found it in the current documentation.

 It would be great to have a list with all the keyboard shortcuts in craft.

Example: 
`CTRL/Command + S` to Save an Entry
`SHIFT + CTRL/Command + S` to Save and Add Another

Found in: https://github.com/craftcms/cms/issues/2851

**Comment by AugustMiller** [comment, marked as answer]
I've included a “Shortcuts” section on the new [Control Panel](https://craftcms.com/docs/4.x/control-panel.html) documentation page. To my knowledge, there aren’t many more non-standard shortcuts (beyond what is possible with keyboard navigation), but that will be where they're cataloged as they're implemented!
